recipes/libical/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "libical" %}
+{% set version = "3.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7ad550c8c49c9b9983658e3ab3e68b1eee2439ec17b169a6b1e6ecb5274e78e6
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage('libical', max_pin='x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - cmake >=3.11.0
+    - perl
+    - pkg-config  # [osx]
+  host:
+    - icu
+    - glib >=2.32
```

Comment:
Remove the version constraint here so that pinning works.

Reply:
I am not sure I understand.  Can you elaborate?

Reply:
do you mean the pins coming from run_exports in those packages?

Reply:
I mean pins coming from `conda-forge-pinning`'s `conda_build_config.yaml`

Reply:
In `conda_build_config.yaml` if you have,
```yaml
glib:
  - 2.56
```

then, following,
```yaml
host:
  - glib
run:
  - glib
```

will be rendered as
```yaml
host:
  - glib  2.56.2
run:
  - glib >=2.56.2,<2.57.0.a0
```


following,
```yaml
host:
  - glib >=2.32
run:
  - glib >=2.32
```

will be rendered as
```yaml
host:
  - glib  2.58.3
run:
  - glib >=2.58.3,<2.59.0a0
```